langmeta/langmeta/shared/src/main/scala/org/langmeta/semanticdb/internal/package.scala

```diff
@@ -10,132 +10,192 @@ import org.langmeta.inputs.{Input => dInput}
 import org.langmeta.inputs.{Position => dPosition}
 import org.langmeta.semanticdb.{Synthetic => dSynthetic}
 import org.langmeta.internal.io.PathIO
-import org.langmeta.internal.semanticdb.{schema => s}
 import org.langmeta.internal.semanticdb.{vfs => v}
 import org.langmeta.io._
 import org.langmeta.semanticdb.Signature
 import org.langmeta.{semanticdb => d}
+import scala.meta.internal.{semanticdb3 => s}
 
 package object semanticdb {
```

Comment:
Unrelated to this PR. I wish all of this logic was outside of the package object, looking for "package.scala" in the scalameta repo yields dozens of results which makes it difficult to find this important file. I don't mind keeping the `implicit class`, but it would be nice to have the logic elsewhere

Reply:
Yes, this whole conversion business is a mess. I'm looking forward to the times when we'll finally have migrated semanticdb-scalac (as well as all Scalameta users) to be based on the low-level API.